dist/screeps.d.ts

```diff
@@ -1017,8 +1017,7 @@ interface SignDefinition {
 }
 interface StoreDefinition {
     [resource: string]: number;
-    energy?: number;
-    power?: number;
+    energy: number;
```

Comment:
Is this actually the case, is energy defined as true all the time? For that matter, is it `[resource: string]: number` or should it be `[resource: string]: number | undefined`?